src/main/java/io/neonbee/endpoint/metrics/MetricsEndpoint.java

```diff
@@ -0,0 +1,32 @@
+package io.neonbee.endpoint.metrics;
+
+import static io.neonbee.endpoint.Endpoint.createRouter;
+
+import java.util.List;
+
+import io.neonbee.config.EndpointConfig;
+import io.neonbee.endpoint.Endpoint;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.micrometer.PrometheusScrapingHandler;
+
+public class MetricsEndpoint implements Endpoint {
+    /**
+     * The default path the metrics endpoint is exposed by NeonBee.
+     */
+    public static final String DEFAULT_BASE_PATH = "/metrics/";
+
+    @Override
+    public EndpointConfig getDefaultConfig() {
+        // as the EndpointConfig stays mutable, do not extract this to a static variable, but return a new object
+        return new EndpointConfig().setType(MetricsEndpoint.class.getName()).setBasePath(DEFAULT_BASE_PATH)
+                // by default the metrics endpoint does not require authentication
```

Comment:
I see this differently. Exposing metrics to everyone is in my opinion a non necessary security issue. Depending on the exposed metrics, someone could figure out what helps best to bring down a service. Or it could even motivate people to bring the service to the limit, because they see live the pressure on the system.

My suggestion: It should be secured by the default authchain by default